WhatsApp/app/src/main/java/com/example/whatsapp/base/BaseActivity.kt

```diff
@@ -50,4 +55,12 @@ abstract class BaseActivity<Binding : ViewBinding> : AppCompatActivity() {
 
     open fun initViews() {}
 
+    private fun applySettings() {
```

Comment:
nit: applySettings seems a bit vauge considering what the function is currently doing.

Reply:
Yup that would seem more appropriate name